Dockerfile

```diff
@@ -21,9 +24,15 @@ RUN sh -c "case $(arch) in aarch64) go_arch=arm64 ;; arm*) go_arch=arm ;; i386|i
 
 COPY config.docker.yml $CONF_DIR/config.yml
 
+COPY entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
+
+VOLUME $LOG_DIR
+
 VOLUME $OUTPUT_DIR
 
 EXPOSE $PORT
 
-ENTRYPOINT ["/usr/bin/bililive-go"]
-CMD ["-c", "/etc/bililive-go/config.yml"]
+WORKDIR /opt/bililive
```

Comment:
改这里打破了至今为止的默认目录，可能导致用户的文件被输出到意料之外的地方，但却看不到什么好处。
如果想修改默认的log输出目录，可以修改 docker 的[默认配置](https://github.com/hr3lxphr6j/bililive-go/blob/master/config.docker.yml)，加入类似
```
log:
  out_put_folder: /opt/bililive
```
的内容。

Reply:
Because the default log address is `/`, root privileges are required to output the address in this location

Reply:
The workdir of the original docker image is root. root directories require root privileges to write to. The purpose of creating a new directory is similar to the concept of a software home, where all software-related files can be placed with user-specified permissions. This way you can better protect your computer in case the software has some security vulnerabilities.

Reply:
在 docker 容器里的话，我们把所有程序相关文件都放进 `/srv/bililive` 里，改成 `WORKDIR /srv/bililive` 如何？
就是不创建 `LOG_DIR`，把默认工作目录放在 `/srv/bililive`，输出 log 也输出到这里。这也符合非 docker 版本的 log 默认输出到视频输出目录，也就是都用 `./` 时的行为。